pilot/pkg/config/kube/gateway/backend_policies.go

```diff
@@ -93,30 +107,22 @@ func DestinationRuleCollection(
 	trafficPolicyStatus, backendTrafficPolicies := BackendTrafficPolicyCollection(trafficPolicies, references, opts)
 	status.RegisterStatus(c.status, trafficPolicyStatus, GetStatus)
 
-	tlsPolicyStatus, backendTLSPolicies := BackendTLSPolicyCollection(tlsPolicies, references, opts)
+	// TODO: BackendTrafficPolicy should also probably use ancestorCollection. However, its still up for debate in the
+	// Gateway API community if having the Gateway as an ancestor ref is required or not; we would prefer it to not be if possible.
```

Comment:
We prefer it not be the Gateway because...? Computation cost? We want Service or route to be the ancestor?